tools/build_framework.sh

```diff
@@ -34,6 +66,23 @@ _notify_github() {
     GIT_REPOSITORY_ROOT=$REPO_ROOT_DIR ${TOOLS_DIR}/github_update.py $1 build:${FRAMEWORK_NAME} $2
 }
 
+# Used below in-order, but here for cli-only
+build_cli() {
+    # CLI (Go):
+    # /home/user/dcos-commons/frameworks/helloworld/cli => frameworks/helloworld/cli
+    REPO_CLI_RELATIVE_PATH="$(echo $CLI_DIR | cut -c $((2 + ${#REPO_ROOT_DIR}))-)"
+    ${TOOLS_DIR}/build_cli.sh ${CLI_EXE_NAME} ${CLI_DIR} ${REPO_CLI_RELATIVE_PATH}
+    if [ $? -ne 0 ]; then
+        _notify_github failure "CLI build failed"
```

Comment:
_notify_github is still living with this code but we now have that functionality handled by plugins. Ideally, that should always be handled by the plugins or on a higher level just so we don't conflate build scripts and our in house CI+review process.

Anyway, my suggestion is that we should probably work towards purging this and now that it doesnt actually functionally hook up to anything, whenever we touch something like this we can just let it disappear. Definitely not a big deal if it stays though